File: Scripts/newTask/factorial/c/factorial.c
#include <stdio.h>

int factorial(int n);

int main() {

for (int i = 0; i < 1000000; ++i) {
int result = factorial(1000);
}

return 0;
}

int factorial(int n) {
    return n == 0 ? 1 : n * factorial(n - 1);
}

File: Scripts/newTask/factorial/go/factorial.go
package main


func factorial(x uint) uint {
	if x == 0 {
		return 1
	}
	
	return x * factorial(x-1)
}

func main() {
 	for i := 0; i < 1000000; i++ {
	x := uint(1000)
	factorial(x)
	}
}
